chore: remove documentation markers from various contracts and files

critesjosh opened the pull request:
This commit cleans up the codebase by removing unnecessary documentation markers from multiple contracts and source files, enhancing readability and maintainability.

sklppy88 reviewed (approved):
The refactor + new docs examples means that these tags only need to exist in the docs examples folder I guess. Nice !

critesjosh committed:
chore: remove documentation markers from various contracts and files

This commit cleans up the codebase by removing unnecessary documentation markers from multiple contracts and source files, enhancing readability and maintainability.
